src/app/pages/form-validation/form-validation.component.html

```diff
@@ -131,7 +131,7 @@ <h2>Password Validation</h2>
                         <mat-icon>{{ newPasswordVisible ? 'visibility' : 'visibility_off' }}</mat-icon>
                     </button>
                 </mat-form-field>
-                <mat-list style="margin: -40px 0 16px;" class="validation-list">
+                <mat-list style="margin: -48px 0 32px;" class="validation-list">
```

Comment:
space between fields should be 32 and switch to 24 below 600px screen width

Reply:
Does it have to be strictly 600px, or can it change with our drawer variant?

Reply:
The drawer variant is being changed to 600 too in another story I think

Reply:
Perfect, this can be updated when our variants update.